### DISCUSSION TITLE
Reading PCE games

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#771] [Need help]
I have the adapter and the menu item enabled for PCE/TG-16. However, I'm a bit confused by the options and can't seem to find a thing searching online. I'm given three options - I know 'TurboChip' is a Turbografx-16 game, but the other two options - HuCard (swapped) and HuCard (not swapped) - what do they mean and how do I know which to pick? Is there any chance of damage to anything if the wrong one is selected?

**Comment by Cephiros** [comment]
No damage. Worst case the rom will dump backwards. I believe that Turbochip, and HuCard(swapped) are the same thing,  Hucard(not swapped) will read the adaptor based on pinout, and the other two will reverse the bit order. Depending on the wiring of your adaptor, one will read TurboGrafx 16 games, and the other will read PC Engine games.

**Reply by PsychoFox11** [reply]
Yeah it was the presence of 3 options that threw me off. I figure there's gotta be a difference in swapped and Turbochip or they wouldn't list them as seperate options. I don't think there was ever a case for ones with the US pin order to be released in Japan. I do know that my TG-16 games say HuCard on them though. Maybe Turbochip was just on the packaging? I don't recall seeing that word on games ever.
Still I think if they were identical, it would be one option, Turbochip/Swapped. Now I gotta figure out the difference!
